projects/concurrency/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import "fmt"
+
+type Value[K comparable] struct {
+	data any
```

Comment:
Rather than `any`, you may want to add a new generic type for the value as well, so that you don't need to make type assertions when retrieving the data.